components/dropdown/basic/src/index.scss

```diff
@@ -97,6 +97,7 @@
       }
       @include media-breakpoint-up(m) {
         background-color: $bgc-dropdown-menu-hover;
+        border-radius: $bdr-dropdown-basic-menu;
```

Comment:
heads up! `$bdr` is border right, please use `$bdrs` for border-radius

Reply:
Yes thanks, already changed in sui theme and in this file 👍